src/main/java/org/apache/mesos/offer/MesosResourcePool.java

```diff
@@ -219,7 +251,8 @@ private boolean sufficientValue(Value desired, Value available) {
         return mesRsrcs;
     }
 
-    private Map<String, MesosResource> getReservedPool(Offer offer) {
+    private static Map<String, MesosResource> getReservedPool(
+            Collection<MesosResource> mesosResources) {
```

Comment:
The mesosResources member was removed as it was effectively only being used for initialization within the constructor